src/main/java/uk/nhs/hee/tis/trainee/forms/dto/LtftFormDto.java

```diff
@@ -58,11 +58,11 @@ public record LtftFormDto(
     UUID id,
     String traineeTisId,
 
-    @JsonView(ReadOnly.class)
+    @JsonView({Views.ReadOnly.class, Views.Admin.class, Views.Trainee.class})
```

Comment:
Is there a reason to list all the views here, and in other places? Afaik if you simply omit the @JsonView annotation it will include the field in all views. Or is there a plan to add other views or otherwise refine the definitions that make this worthwhile?

Reply:
Ah ok, there was a ReadOnly view it needed to belong to, gotcha.